packages/components/src/storyUtils/clickableStyleUtils.tsx

```diff
@@ -298,7 +298,11 @@ const getVariantWithStates = (
                     variant={variant}
                     state={state}
                     disabled={state === "disabled"}
-                    href={tag === "a" ? "" : undefined}
+                    href={tag === "a" ? "www.example.com" : undefined}
+                    onClick={(event: React.MouseEvent<HTMLElement>) => {
```

Comment:
Can TypeScript figure out the type of `event` without the explicit annotation here?

```suggestion
                    onClick={(event) => {
```

Reply:
Guess not; I get
```
➤ YN0000: src/storyUtils/clickableStyleUtils.tsx(302,31): error TS7006: Parameter 'event' implicitly has an 'any' type.
```